### ISSUE TITLE
User cannot focus or select radio button component with keyboard

### ISSUE THREAD

**Issue by dakahn** [carbon-design-system/carbon#1475]
## Environment
Chrome 70
macOS Mojave

## Violation
The user cannot focus or activate a radio button in the component example. Neither space or enter selects or deselects the radio button.

**Comment by dakahn** [comment]
Closing. Selection with the keyboard is done with the arrow keys. 💯
